examples/lambda.test.js

```diff
@@ -0,0 +1,16 @@
+const { WithAwsBackend } = require('../dist');
+
+describe('(Integration:Lambda)', () => {
+	it('should get a response from a lambda', async () => {
```

Comment:
Should this be formatted with prettier too? Think it's tabs and the rest of the project is spaces